### PR TITLE
chore: rennovate bot fixes

### PR DESCRIPTION
**Pull request by daniel-sanche**
- added conventional commit styling to rennovate bot
- reduced number of PRs to 3
  - 5 seems to overwhelm the e2e tests when they all rebase at once

### CODE REVIEW

**Comment by minherz** [comment]
LGTM

**Review by minherz — APPROVED** [review]
lgtm

**Review by simonz130 — APPROVED** [review]

File: renovate.json [hunk 1]
```diff
@@ -28,5 +28,6 @@
   },
   "ignoreDeps": [
     "com.google.protobuf:protoc"
-  ]
+  ],
+  "semanticCommits" : "enabled"
```
Comment (simonz130) [inline renovate.json:32]: I'm not sure if semanticCommits = conventional commits, but at least they are very close

Comment (daniel-sanche) [reply renovate.json]: yeah, it looks like they append "chore" to their commits, which is what we'd want

**Pull request closed without merging**